src/compiler/scanner.ts

```diff
@@ -829,7 +829,7 @@ namespace ts {
             getToken: () => token,
             getTokenPos: () => tokenPos,
             getTokenText: () => text.substring(tokenPos, pos),
-            getTokenValue: () => tokenValue,
+            getTokenValue: () => Debug.assertDefined(tokenValue),

```

Comment:
please add a message to the assert.

Reply:
i would not put an assert in the scanner.. if this triggeres the whole LS will be broken.